src/models/event-timeline.ts

```diff
@@ -24,6 +24,30 @@ import { MatrixEvent } from "./event";
 import { Filter } from "../filter";
 import { EventType } from "../@types/event";
 
+export interface IInitialiseStateOptions {
+    /** Whether the timeline was empty before the marker arrived in
+     *  the room. This could be happen in a variety of cases:
+     *  1. From the initial sync
+     *  2. It's the first state we're seeing after joining the room
+     *  3. Or whether it's coming from `syncFromCache` */
```

Comment:
We have 4 copies of this exact comment, and its kinda vague to the reader `the marker` doesn't mean much in isolation.

Reply:
More context in https://github.com/matrix-org/matrix-js-sdk/pull/2299#discussion_r883502814

Addressed vague `marker` in https://github.com/matrix-org/matrix-js-sdk/pull/2299#discussion_r883499258